contributor/outline.md

```diff
@@ -1,64 +1,138 @@
-   - If your contribution is large, be sure to start queuing it up early enough so that it's "in" by the time that you need it
-   - Remember that everyone is working together to do their best, leave any sense of entitlement or privilege behind when interacting with the host team.
+  - Introduce the speaker.
+  - Quick overview of the _Contributor_ role.
+  - Summary of the roles of InnerSource and how each relates to _Contributor_.
+  - Summary of the upcoming segments and what each covers.
 
-## Technical Concerns
-   - Read the CONTRIBUTING.md and follow it.
-    - Keep in mind larger timezone differences when managing your expectations for reaction times.
-   - If you are contributing the fix to an issue, be sure to follow all guidance and instruction documented in the issue.
-   - Update any documentation that goes along with your code.
-   - Take feedback as data that helps you refine your idea/code
-   - Even after merge, if the Host team has any questions or additional help needed from  you for them to understand and maintain what you contributed, be timely and responsive.
-    - the '30 days warranty' pattern appears to be relevant here. Otherwise given the change of teams in person a "fear of maintenance" and the matching "fear to consume iss artifacts" can ensue.
-   - If you are stuck while working on a contribution, then check the project documentation for ways to get support.
-   Don't spin or give up on your valuable addition.
-   - Behave politely in pull request.
-   - Comments - contributions should be documented.
-   - Fixes for typos and builds are great contributions.
-   - Style of code should be similar to what code is already present.
+Outline for this section inspired by the [Trusted Committer introduction video].
 
-   - How acceptable / feasible / desired are upstream connected forks that work in a stable/experimental alternative setup? 
-    - The OSS environment has such forks where the experimental fork accepts more than the stable fork but there is exchange/upstreaming of proven experimental forks. (e.g. Psi/Psi+ or possibly mainline/LTS Linux)
-    - In an Inner Source environment this might be less feasible due to a smaller amount of resources, however might make a pure research / r&d / maintenance transfer easier
+## Becoming an InnerSource Contributor
 
+  - A contributing mindset starts with a shared use mindset.
+    - When you have technical needs that are outside of the domain of your project,
+    look around to see if there is a shared solution that you can use.
+    - Briefly enumerate the benefits of using shared solutions.
+  - Using shared solutions puts you in a position to become a _Contributor_.
+    - There will be times when you wish that the shared solution has some new feature.
+    - Don't _Wait it Out_ or _Work Around_, spend your time and energy making the change directly in the shared project.
```

Comment:
Nice re-use of the Antipatterns. :) (We should probably also start writing them down as we do with the patterns I guess.)